samples/crypto/kmu_usage_nrf54l/README.rst

```diff
@@ -0,0 +1,149 @@
+.. _crypto_kmu_usage_nrf54l:
+
+Crypto: KMU usage (nRF54L series)
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The KMU usage sample shows how to generate and use various persistent keys using the Platform Security Architecture (PSA) APIs.
+Persistent keys are stored in the Key Management Unit (KMU) of the device and retain their value between resets.
+The implementation of the PSA ITS API is provided by the Oberon PSA Crypto, which implements the driver wrapper role
+and determines which driver to use for each cryptographic operation.
+For this sample, CRACEN driver is used.
+
+The sample is also used to demonstrate the usage of the :ref:`pre-provisioned keys <ug_nrf54l_developing_provision_kmu_provisioning>`.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54l15dk_nrf54l05_cpuapp, nrf54l15dk_nrf54l10_cpuapp, nrf54l15dk_nrf54l15_cpuapp, nrf54l15dk_nrf54l15_cpuapp_ns, nrf54lm20dk_nrf54lm20a_cpuapp, nrf54lv10dk_nrf54lv10a_cpuapp
+
+.. include:: /includes/tfm.txt
+
+Overview
+********
+
+In this sample, four different keys/keypairs are used:
```

Comment:
This sentence says four, but you only list three keys?

Reply:
Addressed.